templates/abstrack.html

```
@@ -54,7 +61,7 @@ <h5 class="card-title">{{ _("Track Info:") }}</h5>
                     <strong>{{ _("Link:") }}</strong>
                     <a href="{{ track.track_share_url.split(" ?")[0] }}"
                        class="card-link"
-                       target="_blank">Musixmatch</a> |
+                       target="_blank">{{ _(Musixmatch) }}</a> |
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)